### ISSUE TITLE
Allow to configure social links

### ISSUE THREAD

**Issue by egoist** [egoist/demoboard#19]
- twitter
- github
- donate
- custom

**Comment by gurrrung** [comment]
Hey, @egoist, can I take it? :) A little more details about your expectations would be helpful too.

**Comment by egoist** [comment]
@gurrrung Yeah but I'm not sure where is the best place to put those links.

**Comment by gurrrung** [comment]
@egoist what purpose do they solve?

**Comment by egoist** [comment]
@gurrrung As Demoboard is used to build demos, usually you want to add links to your original GitHub repository and your Twitter page etc.

**Comment by gurrrung** [comment]
How is the bottom section of the sidebar?

**Comment by egoist** [comment]
Sounds good, I can imagine a fixed bottom section 👌

**Comment by egoist** [comment]
https://feathericons.com and https://akveo.github.io/eva-icons are great source for svg icons.

**Comment by saltysugar** [comment]
I'm working on this.

**Comment by kenanchristian** [comment]
I saw that this issue hasn't been updated for 20 days, so I tried giving it a shot.
